src/util/time.cpp

```diff
@@ -151,6 +151,19 @@ std::string FormatISO8601DateTime(int64_t nTime) {
     return strprintf("%04i-%02i-%02iT%02i:%02i:%02iZ", ts.tm_year + 1900, ts.tm_mon + 1, ts.tm_mday, ts.tm_hour, ts.tm_min, ts.tm_sec);
 }
 
+std::string FormatISO8601DateTimeDashSep(int64_t nTime) {
```

Comment:
function name is a little wordy, but looks fine otherwise

Reply:
Wordy is better than cryptic :)